### DISCUSSION TITLE
Making app accessible by registered users but not public

### DISCUSSION THREAD

**Discussion by rqi14** [Budibase/budibase#12211] [Help] [answered]
Hi. Is there a way to make an app having an accessible level for all login users but not public. Or on the landing page to ask the users to login?

**Comment by ConorWebb96** [comment]
Hey @rqi14,

Any app thats published with basic or above permissions will redirect the user to the login screen.

I hope this helps.

**Reply by rqi14** [reply]
Hi. Thanks for the prompt reply. Setting it basic means one has to give the user permission to use the app. What I meant is a way to make an app available to all registered users without needing to manually give users permission.

**Reply by ConorWebb96** [reply, marked as answer]
Hey @rqi14,

Ahh I understand now. Outside of using [User Groups](https://docs.budibase.com/docs/user-groups), I don't believe there is a way to achieve this. With User groups you can create different departments which have their own permissions meaning you will only have to setup it once and any changes are done to the group instead of the users themselves.

I hope this helps.

**Reply by rqi14** [reply]
Understood. It will be very helpful to have a functionality like this. Now there are public and basic access levels, but there is not anything in between for normal registered users. It is rather weird, isn’t it?
